tests/driver/cpp/build.rs

```diff
@@ -51,6 +51,12 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             continue;
```

Comment:
it would be nice if we could add something like `//ignore: cpp` into test cases but then we'd have to read test case sources at build time in the test drivers.